SteamReview/SteamReview/Steam.cpp

```diff
@@ -8,6 +8,14 @@ FSteam::FSteam()
 	ActiveMenuCommand = EMenuCommand::MainMenu;
 };
 
+//---------------------------------------------------------
+FSteam::FSteam(FSteam& OtherSteam)
```

Comment:
Remember that you want to use _const_ references if the original value isn't supposed to change.